couch/check.py

```diff
@@ -119,7 +120,7 @@ def get_data(self, server, instance):
             databases = list(databases)[:self.MAX_DB]
 
         for dbName in databases:
-            url = urljoin(server, dbName)
+            url = urljoin(server, quote(dbName, safe = ''))
```

Comment:
OK, so I just tested this manually. Nothing to worry about, unicode names for databases are apparently not supported:
```
{"error":"illegal_database_name","reason":"Name: 'kennel/こんにちは・今日は'. Only lowercase characters (a-z), digits (0-9), and any of the characters _, $, (, ), +, -, and / are allowed. Must begin with a letter."}
```

Also, the PR works as advertised. Merging.